Update tr-tr.xaml

crasadure opened the pull request:
new added lines is translated to turkish language

YerongAI commented on src/OfficeToolPlus/Dictionaries/Languages/tr-tr.xaml:
```diff
@@ -132,17 +132,17 @@
+    <sys:String x:Key="Deploy_iSlide">Yer tutucu</sys:String>
+    <sys:String x:Key="Deploy_iSlide.Description">YerTutucu</sys:String>
+    <sys:String x:Key="Deploy_iSlide_Install">YerTutucu</sys:String>
+    <sys:String x:Key="Deploy_iSlide_Install.Name">YerTutucu</sys:String>
```
Line 142 to 145 is no need to translate.

YerongAI commented on src/OfficeToolPlus/Dictionaries/Languages/tr-tr.xaml:
```diff
@@ -298,70 +298,70 @@
+    <sys:String x:Key="Activate_License.Description">Kurulu lisansları yönetin.</sys:String>
+    <sys:String x:Key="Activate_License_Refresh">_Yenile</sys:String>
+    <sys:String x:Key="Activate_License_Expand">_Hepsini genişlet</sys:String>
+    <sys:String x:Key="Activate_License_Collapse">_Hepsini Daralt</sys:String>
```
The access key is same as previous one, please select others one. If you don't know the access key, please visit: https://wpf-tutorial.com/control-concepts/access-keys/

YerongAI commented on src/OfficeToolPlus/Dictionaries/Languages/tr-tr.xaml:
```diff
@@ -686,7 +686,7 @@
     <!-- Learn more about hotkey and autorun.inf: https://en.wikipedia.org/wiki/Autorun.inf#:~:text=shell%5Cverb%3Dmenu%20text -->
     <!-- If you can't understand what is hotkey, remove '&amp;' when you are translating. -->
-    <sys:String x:Key="CD-ROM_InstallNow">&amp;Install now</sys:String>
+    <sys:String x:Key="CD-ROM_InstallNow">&amp;Şimdi Yükle</sys:String>
```
The access key shoule be `I`, recommend to change to `Şimd&amp;i Yükle`

YerongAI commented on src/OfficeToolPlus/Dictionaries/Languages/tr-tr.xaml:
```diff
@@ -756,21 +756,21 @@
     <sys:String x:Key="Info_KMS_Test_Result.Success">KMS test yöntemi: {0}. Test sonucu: başarı, geçen süre: {1}ms.</sys:String>
     <sys:String x:Key="Info_KMS_Test_Result.Failed">KMS test hatası.</sys:String>
-    <sys:String x:Key="Info_Parallel_Download">You are using 32-bit version of Office Tool Plus, enable "Parallel download in chunks" may cause out of memory error.&#x000A;&#x000A;If application crashes during download, please disable "Parallel download in chunks" or switch to 64-bit version of Office Tool Plus to avoid the problem.</sys:String>
+    <sys:String x:Key="Info_Parallel_Download">Office Tool Plus'ın 32 bit sürümünü kullanıyorsanız, "Parçalar halinde paralel indirme" özelliğini etkinleştirmek yetersiz bellek hatasına neden olabilir.& # x000A;& # x000A; İndirme sırasında uygulama çökerse, sorunu önlemek için lütfen "Parçalar halinde paralel indirmeyi" devre dışı bırakın veya Office Tool Plus'ın 64 bit sürümüne geçin.</sys:Strin ...
```
Do not change `&#x000A;&#x000A;`, `& # x000A;& # x000A;` is wrong text.

YerongAI commented on src/OfficeToolPlus/Dictionaries/Languages/tr-tr.xaml:
```diff
@@ -797,18 +797,18 @@
+    <sys:String x:Key="Err_Act_Office_License">Ürün etkinleştirilirken bir hata oluştu.& #x000A; &#x000A; Hata kodu: {0}</sys:String>
+    <sys:String x:Key="Err_Install_Office_Lic">Lisansı yüklerken bir hata oluştu.& #x000A; &#x000A; Hata kodu: {0}</sys:String>
+    <sys:String x:Key="Err_Install_Product_Key">Ürün anahtarını yüklerken bir hata oluştu.& #x000A; &#x000A; Hata kodu: {0}</sys:String>
+    <sys:String x:Key="Err_Uninstall_Product_Key">Ürün anahtarı kaldırılırken bir hata oluştu.& #x000A; &#x000A; Hata kodu: {0}</sys:String>
```
Do not change `&#x000A;&#x000A;`, `& # x000A;& # x000A;` is wrong text.

YerongAI reviewed (changes requested):
Something wrong, please change them. If any questions please let us know.

crasadure commented:
I fixed all the errors you pointed out as you described. I would appreciate if you let me know if there are any other mistakes. Thank you for the information.

I updated github tr-tr.xaml file.
________________________________
Gönderen: Yerong ***@***.***>
Gönderildi: 23 Nisan 2023 Pazar 05:32
Kime: YerongAI/Office-Tool ***@***.***>
Bilgi: Tayfun Akkoyun ***@***.***>; Author ***@***.***>
Konu: Re: [YerongAI/Office-Tool] Update tr-tr.xaml (PR #676)


@YerongAI requested changes on this pull request.

Something wrong, please change them. If any questions please let us know.

________________________________

In src/OfficeToolPlus/Dictionaries/Languages/tr-tr.xaml<https://github.com/YerongAI/Office-Tool/pull/676#discussion_r1174494621>:

>      <sys:String x:Key="Deploy_Module.OTP">Office Tool Plus</sys:String>
     <sys:String x:Key="Deploy_Download_First">Önce indir, sonra kur</sys:String>
     <sys:String x:Key="Deploy_Download_First.Description">Kurulum dosyaları, program dizinine veya kaynak yolu ile belirtilen dizine kaydedilecektir.</sys:String>
     <sys:String x:Key="Deploy_Create_Shortcut">Masaüstü kısayolları oluştur</sys:String>
     <sys:String x:Key="Deploy_Create_Shortcut.Description">Office yüklemesinden sonra Office uygulamalarına masaüstü kısayolları oluştur.</sys:String>
     <!-- Do not edit it or translate it. iSlide only provided in mainland China. -->
-    <sys:String x:Key="Deploy_iSlide">PlaceHolder</sys:String>
-    <sys:String x:Key="Deploy_iSlide.Description">PlaceHolder</sys:String>
-    <sys:String x:Key="Deploy_iSlide_Install">PlaceHolder</sys:String>
-    <sys:String x:Key="Deploy_iSlide_Install.Name">PlaceHolder</sys:String>
+    <sys:String x:Key="Deploy_iSlide">Yer tutucu</sys:String>
+    <sys:String x:Key="Deploy_iSlide.Description">YerTutucu</sys:String>
+    <sys:String x:Key="Deploy_iSlide_Install">YerTutucu</sys:String>
+    <sys:String x:Key="Deploy_iSlide_Install.Name">YerTutucu</sys:String>


Line 142 to 145 is no need to translate.

________________________________

In src/OfficeToolPlus/Dictionaries/Languages/tr-tr.xaml<https://github.com/YerongAI/Office-Tool/pull/676#discussion_r1174494812>:

> @@ -298,70 +298,70 @@
     <sys:String x:Key="Activate_KMS_Reset.Description">KMS sunucu adını kaldırın ve KMS bağlantı noktasını varsayılan olarak ayarlayın.</sys:String>

     <!-- Licenses list -->
-    <sys:String x:Key="Activate_Licenses">Office licenses</sys:String>
-    <sys:String x:Key="Activate_License">Licenses list</sys:String>
-    <sys:String x:Key="Activate_License.Description">Manage installed licenses.</sys:String>
-    <sys:String x:Key="Activate_License_Refresh">_Refresh</sys:String>
-    <sys:String x:Key="Activate_License_Expand">_Expand all</sys:String>
-    <sys:String x:Key="Activate_License_Collapse">_Collapse all</sys:String>
-    <sys:String x:Key="Activate_License_Clear">C_lear list</sys:String>
-    <sys:String x:Key="Activate_License_Status.Unactivated">Unactivated</sys:String>
-    <sys:String x:Key="Activate_License_Status.Activated">Activated</sys:String>
+    <sys:String x:Key="Activate_Licenses">Ofis lisansları</sys:String>
+    <sys:String x:Key="Activate_License">Lisans listesi</sys:String>
+    <sys:String x:Key="Activate_License.Description">Kurulu lisansları yönetin.</sys:String>
+    <sys:String x:Key="Activate_License_Refresh">_Yenile</sys:String>
+    <sys:String x:Key="Activate_License_Expand">_Hepsini genişlet</sys:String>
+    <sys:String x:Key="Activate_License_Collapse">_Hepsini Daralt</sys:String>


The access key is same as previous one, please select others one. If you don't know the access key, please visit: https://wpf-tutorial.com/control-concepts/access-keys/

________________________________

In src/OfficeToolPlus/Dictionaries/Languages/tr-tr.xaml<https://github.com/YerongAI/Office-Tool/pull/676#discussion_r1174494883>:

> @@ -686,7 +686,7 @@
     <!-- What is &amp; means? '&amp;' stands for '&', so it is &Install now, users can press the hotkey (I) to execute the command when menu is opened. -->
     <!-- Learn more about hotkey and autorun.inf: https://en.wikipedia.org/wiki/Autorun.inf#:~:text=shell%5Cverb%3Dmenu%20text -->
     <!-- If you can't understand what is hotkey, remove '&amp;' when you are translating. -->
-    <sys:String x:Key="CD-ROM_InstallNow">&amp;Install now</sys:String>
+    <sys:String x:Key="CD-ROM_InstallNow">&amp;Şimdi Yükle</sys:String>


The access key shoule be I, recommend to change to Şimd&amp;i Yükle

________________________________

In src/OfficeToolPlus/Dictionaries/Languages/tr-tr.xaml<https://github.com/YerongAI/Office-Tool/pull/676#discussion_r1174494949>:

>      <sys:String x:Key="Info_KMS_Test_Result.Success">KMS test yöntemi: {0}. Test sonucu: başarı, geçen süre: {1}ms.</sys:String>
     <sys:String x:Key="Info_KMS_Test_Result.Failed">KMS test hatası.</sys:String>
-    <sys:String x:Key="Info_Parallel_Download">You are using 32-bit version of Office Tool Plus, enable "Parallel download in chunks" may cause out of memory error.&#x000A;&#x000A;If application crashes during download, please disable "Parallel download in chunks" or switch to 64-bit version of Office Tool Plus to avoid the problem.</sys:String>
+    <sys:String x:Key="Info_Parallel_Download">Office Tool Plus'ın 32 bit sürümünü kullanıyorsanız, "Parçalar halinde paralel indirme" özelliğini etkinleştirmek yetersiz bellek hatasına neden olabilir.& # x000A;& # x000A; İndirme sırasında uygulama çökerse, sorunu önlemek için lütfen "Parçalar halinde paralel indirmeyi" devre dışı bırakın veya Office Tool Plus'ın 64 bit sürümüne geçin.</sys:String>


Do not change &#x000A;&#x000A;, & # x000A;& # x000A; is wrong text.

________________________________

In src/OfficeToolPlus/Dictionaries/Languages/tr-tr.xaml<https://github.com/YerongAI/Office-Tool/pull/676#discussion_r1174495075>:

>      <sys:String x:Key="Err_Loading_Office_Licenses">Bu bilgisayarda Office'in Tıkla-Çalıştır sürümü algılanmadı, Office lisansları yüklenemedi. Lütfen Office'in en son sürümünün yüklü olduğundan emin olun.</sys:String>
     <sys:String x:Key="Err_Product_Key_Invalid">Girilen ürün anahtarı geçersiz, lütfen kontrol edip tekrar deneyin.</sys:String>
     <sys:String x:Key="Err_Confirmation_ID_Invalid">Girilen onay kimliği geçersiz, lütfen kontrol edin ve tekrar deneyin.</sys:String>
-    <sys:String x:Key="Err_KMS_Host_Invalid">Unable to save KMS settings, please check the KMS host name and try again.</sys:String>
-    <sys:String x:Key="Err_No_KMS_Lic">Unable to save KMS settings, no Office KMS licenses were installed on the system.</sys:String>
-    <sys:String x:Key="Err_Set_KMS">Unable to save KMS settings, please try again.</sys:String>
-    <sys:String x:Key="Err_Act_Office_License">An error occurred while activating the product.&#x000A;&#x000A;Error code: {0}</sys:String>
-    <sys:String x:Key="Err_Install_Office_Lic">An error occurred while installing the license.&#x000A;&#x000A;Error code: {0}</sys:String>
-    <sys:String x:Key="Err_Install_Product_Key">An error occurred while installing the product key.&#x000A;&#x000A;Error code: {0}</sys:String>
-    <sys:String x:Key="Err_Uninstall_Product_Key">An error occurred while uninstalling the product key.&#x000A;&#x000A;Error code: {0}</sys:String>
-    <sys:String x:Key="Err_Check_KMS_Status">Unable to check KMS status, please check the KMS host name and try again.</sys:String>
+    <sys:String x:Key="Err_KMS_Host_Invalid">KMS ayarları kaydedilemiyor, lütfen KMS ana bilgisayar adını kontrol edin ve tekrar deneyin.</sys:String>
+    <sys:String x:Key="Err_No_KMS_Lic">KMS ayarları kaydedilemedi, sisteme Office KMS lisansı yüklenmedi.</sys:String>
+    <sys:String x:Key="Err_Set_KMS">KMS ayarları kaydedilemiyor, lütfen tekrar deneyin.</sys:String>
+    <sys:String x:Key="Err_Act_Office_License">Ürün etkinleştirilirken bir hata oluştu.& #x000A; &#x000A; Hata kodu: {0}</sys:String>
+    <sys:String x:Key="Err_Install_Office_Lic">Lisansı yüklerken bir hata oluştu.& #x000A; &#x000A; Hata kodu: {0}</sys:String>
+    <sys:String x:Key="Err_Install_Product_Key">Ürün anahtarını yüklerken bir hata oluştu.& #x000A; &#x000A; Hata kodu: {0}</sys:String>
+    <sys:String x:Key="Err_Uninstall_Product_Key">Ürün anahtarı kaldırılırken bir hata oluştu.& #x000A; &#x000A; Hata kodu: {0}</sys:String>


Do not change &#x000A;&#x000A;, & # x000A;& # x000A; is wrong text.

—
Reply to this email directly, view it on GitHub<https://github.com/YerongAI/Office-Tool/pull/676#pullrequestreview-1396822644>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ANAJ7IXJS6NWT76RL4HURDTXCSICHANCNFSM6AAAAAAXIBBEMM>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
